.changeset/tall-planes-nail.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-ts/transactions": patch
+"@fuel-ts/contract": patch
+"@fuel-ts/account": patch
+"@fuel-ts/errors": patch
+---
+
+feat: deploy contract validation
```

Comment:
Should this be a breaking change?

Reply:
Also, following [previous releases](https://github.com/FuelLabs/fuels-ts/releases), it'd be great to stick to short and sweet release notes and use the summary for more lengthy details.

Leaving an idea below, see if you have a better one.

----

In this release, we:
 - Limited deployment of contracts up to 100kb